client/components/CreateJob.jsx

```diff
@@ -95,16 +106,38 @@ export default function CreateJob() {
         <InnerContainer>
           <Header>Post a Job</Header>
           <Form onSubmit={handleSubmit}>
+            <Select onChange={handleChange} value={form.industry} name="industry" required>
+              <option value="" defaultValue>Select Industry *</option>
+              <option value="Agriculture">Agriculture</option>
```

Comment:
Could `map` be used to list out these different options? `var arr = [Agriculture, Construction]; arr.map(() => {})` etc

Reply:
ohh interesting I didn't think of that, I'll do that when I have a chance to refactor